npe2/manifest/schema.py

```diff
@@ -167,9 +164,7 @@ def from_distribution(cls, name: str) -> PluginManifest:
         dist = distribution(name)  # may raise PackageNotFoundError
         for ep in dist.entry_points:
             if ep.group == ENTRY_POINT:
-                pm = PluginManifest._from_entrypoint(ep)
-                pm._populate_missing_meta(dist.metadata)
-                return pm
+                return PluginManifest._from_entrypoint(ep, dist)
```

Comment:
codecov is saying this didn't get hit by tests, but it does get hit on my machine